VideoLocker/src/main/java/org/edx/mobile/player/PlayerFragment.java

```diff
@@ -486,9 +491,6 @@ public synchronized void playOrPrepare(String path, int seekTo, String title,
             exitFullScreen();
         }
 
-        // clear all errors
-        clearAllErrors();
```

Comment:
Should we have `hideProgress()` separated from `clearAllErrors()`, and still clear just errors here? We can have a new `clearState()` method that calls both, and replace other calls to `clearAllErrors()` with it.